cogs/login.py

```diff
@@ -0,0 +1,150 @@
+import json
+import logging
+from pathlib import Path
+from typing import Any, Dict
+
+import discord
+from discord import Color, Embed
+from discord.ext import commands
+from discord.ext.commands.context import Context
+from discord.ui import Button, Select, View                 #type: ignore
+
+from main import ManChanBot
+
+from .commandbase import CommandBase
+
+class Login(CommandBase):
+    def open_json():                                        # type: ignore
+        # Checks to see if File Exists
+        path = "login.json"
+        obj = Path(path)
+
+        if obj.exists():
+            try:
+                # Check to see if JSON File opens properly
+                with open(path) as f:
+                    return json.loads(f.read())
+                # Error decoding JSON File, File Invalid
+            except json.decoder.JSONDecodeError:
+                print("JSON File is not valid")
+                return
+                # Opening Error
+            except IOError:
+                print(f"Error opening {path}")
+                return
+        else:
+            return
+
+    def get_account_info(self, data: Any, site: str) -> str:
+        entries = list(filter(lambda entry: entry["site"] == site, data["login_info"]))
+        return entries[0]
+
+    async def create_selection_embed(self,
+        previous: Any, selection: Any, home: Any, old_view: Any
```

Comment:
The parameter names are confusing, and the fact that they are all Any type doesn't help.

From what I infer:
`previous`: This is a Message type. Name this `current_message` or so.

`selection`: This is a dict object based on the login/account info. Name it so.

`home`: This is an Embed. This coincides with `old_view`, so call this `old_embed`.

`old_view`: This is a View but python is dumb with typing for it (stubs isnt updated for this), so Any is fine